SPECS/sm.spec

```diff
@@ -431,6 +439,14 @@ then
 fi
 
 %changelog
+* Fri Jul 04 2025 Mathieu Labourier <mathieu.labourier@vates.fr> - 3.2.12-3.4
```

Comment:
Isn't it vates.tech?

Reply:
Indeed, fixed.

Reply:
I can only blame myself for the lousy copaste, apologies for that 🙏 It should be all good now.